types/src/execution_result.rs

```diff
@@ -111,13 +111,17 @@ static EXECUTION_RESULT: Lazy<ExecutionResult> = Lazy::new(|| {
 });
 
 /// The result of executing a single deploy.
+// Note: For backward compatibility reasons, the `execution_effect` fields are serialized under the
+// name `effect`. We could consider introducing a json representation struct here as we have done
+// elsewhere for `Block`.
```

Comment:
Ah, I see! Sorry for complicating this!
Maybe then we should use `effect` everywhere instead of `execution_effect`?

Reply:
Fixed in b122856458683d0de58deb3914bda944b9afd05c